.travis.yml

```diff
@@ -32,18 +32,6 @@ matrix:
       - python: 3.5
 
     include:
-    - dist: trusty
-      env:
-        - JOB="3.7" ENV_FILE="ci/deps/travis-37.yaml" PATTERN="not slow and not network"
```

Comment:
leave the main 3.7 one here and pick another (the 3.6 is ok)

Reply:
The problem is that we don't have mysql or postgresql in azure. The ones I moved are the ones that do not run the database tests.

I can leave the main 3.7 in travis, but we'll have the CI unbalanced, and will have to wait for travis (less than now if I move the other 2).

Reply:
moved 3.7 back to travis